python/mage/export_import_util/parameters.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+
+class Parameter(Enum):
+    ID = "id"
+    LABELS = "labels"
+    PROPERTIES = "properties"
+    TYPE = "type"
+    END = "end"
+    LABEL = "label"
+    START = "start"
+    NODE = "node"
+    RELATIONSHIP = "relationship"
+    DURATION = "duration("
+    LOCALTIME = "localTime("
+    LOCALDATETIME = "localDateTime("
+    DATE = "date("
```

Comment:
Are the trailing `)`s required? If they're not, consider removing them.

Reply:
Did you mean `(`? I explained my reasoning in the comment below, but feel free to comment more.

Reply:
Thanks! Marking this as resolved.